yogstation/code/modules/jobs/job_types/mining_medic.dm

```diff
@@ -43,4 +43,5 @@
 	mask = /obj/item/clothing/mask/surgical
```

Comment:
While we are here, can we remove this stupid mask that you can't use because you're meant to be on lavaland with internals anyways?
```suggestion
```